en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/M/Morschecks_Hand.md

```diff
@@ -17,13 +36,14 @@ The spell ends early if it is dropped to 0 hit points.
 When you cast the spell and as a bonus action on subsequent turns, you can move the hand up to 60 feet and then choose one of the following.
 
 - **Shove:**
-  The hand makes a Strength saving throw against the maneuver DC of a creature within 5 feet of it, with advantage if the creature is Medium or smaller.
+  The hand makes a Strength check against a creature within 5 feet of it, contested by a Strength (Athletics) check.
```

Comment:
I'm guessing "maneuver DC" is an A5e concept. We don't have that here, so I used the mechanics from WotC. (Same with the _Snatch_ option.)